server/binding.js

```diff
@@ -84,15 +91,21 @@ binding.query.Locus;
 /**
  * @param {!binding.query.Histogram} query
  * @param {string} bindingPath
- * @return {?binding.Histogram}
+ * @return {?binding.Histogram|binding.Error}
```

Comment:
```js
@return {binding.Histogram|binding.Error}
```
If error handling is involved, there won't be the necessity to return null I think?